packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -209,7 +215,9 @@ export default function(props: Props) {
 			);
 		}
 	}
-
+	/**
```

ESLint

[error] Expected multiple line comments instead of a block comment.

(multiline-comment-style)